---
 Author: Gilles Dartiguelongue <eva@gentoo.org>

 Make warning and deprecation cflags optional, this is especially important for downstream.

 https://bugs.gentoo.org/show_bug.cgi?id=263078

 configure.ac |    9 +++++++++
 1 file changed, 9 insertions(+)

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -21,6 +21,9 @@ AC_PATH_PROG(GLIB_GENMARSHAL, glib-genma
 dnl ---------------------------------------------------------------------------
 dnl - Extra verbose warning switches
 dnl ---------------------------------------------------------------------------
+
+case "$enable_compile_warnings" in
+	yes|maximum|error)
 #CPPFLAGS="$CPPFLAGS -DG_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES"
 CPPFLAGS="$CPPFLAGS -DG_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED"
 #CPPFLAGS="$CPPFLAGS -DGDK_MULTIHEAD_SAFE -DGTK_MULTIHEAD_SAFE"
@@ -59,6 +62,12 @@ else
 	WARNINGFLAGS=""
 	WARNINGFLAGS=""
 fi
+	;;
+	*)
+	WARNINGFLAGS=""
+	WARNINGFLAGS=""
+	;;
+esac
 AC_SUBST(WARNINGFLAGS)
 AC_SUBST(WARNINGFLAGS)
 
